examples/pipelines/mnist.py

```diff
@@ -21,16 +31,19 @@
 It also takes code from
 https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/examples/how_tos/reading_data/convert_to_records.py
 
+This example shows how you can save TensorFlow outputs into bundles.
+
+Pre Execution:
+$export PYTHONPATH=$DISDAT_HOME/disdat/examples/pipelines
+$dsdt context examples; dsdt switch examples
+
+Execution:
+$python ./mnist.py
+or:
+$dsdt apply - - mnist.Evaluate
```

Comment:
This example does not work out of the box. I think `setup.py` needs `tensorflow` in `install_requires`.

Reply:
Or, since Disdat itself doesn't require `tensorflow`, possibly a `pip install` instruction or something...

Reply:
Updated comments to `pip install -e .` in the experiments directory.